app/assets/javascripts/dashboard.js

```diff
@@ -79,8 +80,11 @@ MinionPoller = {
         for (i = 0; i < minions.length; i++) {
           rendered += MinionPoller[renderMethod].call(MinionPoller, minions[i]);
 
-          if (minions[i].highstate != "applied") {
-            allApplied = false;
+          if (minions[i].highstate == "applied") {
+            allApplied = true;
```

Comment:
Ouch, this will set `allApplied` to `true` with only one applied, is this intended? Maybe we can use `Array.prototype.every()` to set `allApplied`